src/Sets/VPolytope.jl

```diff
@@ -555,10 +555,6 @@ function load_polyhedra_vpolytope() # function to be loaded by Requires
 return quote
 # see the interface file AbstractPolytope.jl for the imports
 
-export vertices_list,
-       tohrep,
```

Comment:
`vertices_list` is redundant withing the file, but `tohrep` and `tovrep` where defined only once in `load_polyhedra_vpolytope`. I wouldn't remove these exports from `VPolytope.jl`.

Reply:
These `export`s were defined deep inside the file (look at the line numbers). They should be at the top of the file. I can move them there if you prefer.